sdk/cosmos/microsoft-azure-cosmos-benchmark/pom.xml

```diff
@@ -8,12 +8,12 @@ Licensed under the MIT License.
   <parent>
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-cosmos-parent</artifactId>
-    <version>3.4.0</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmos-parent;current} -->
```

Comment:
Just to double check 3.4.0 was just a snapshot right?

Reply:
Not anymore. 
The practice has been changed to update the versions to next to be released version.